internal/future/serving/server.go

```diff
@@ -60,6 +60,8 @@ type Params struct {
 	publicCertificateFileData []byte
 	// Private key in PEM format.
 	privateKeyFileData []byte
+	// Store is an interface to the state storage used to access Open Match state.
+	Store statestore.Service
```

Comment:
This should not be a part of the server.

Reply:
Just initialize in each service then?